packages/aws-cdk/lib/settings.ts

```diff
@@ -185,6 +187,18 @@ export class Settings {
     const context = this.parseStringContextListToObject(argv);
     const tags = this.parseStringTagsListToObject(expectStringList(argv.tags));
 
+    // Determine bundling stacks
+    let bundlingStacks: string[];
+    if (BUNDLING_COMMANDS.includes(argv._[0])) {
+    // If we deploy, diff or synth a list of stacks exclusively we skip
+    // bundling for all other stacks.
+      bundlingStacks = argv.exclusively
```

Comment:
This stringly-typed coupling to the `yargs` configuration in a completely different file is making me very nervous.

Can we at least add these to the definition of `Arguments`?

Reply:
If I type the first element of `_`,

```ts
export type Arguments = {
  readonly [name: string]: unknown,
  readonly _: [Command, ...string[]],
};
```

Then I'm not sure how I can make the TS compiler happy here?
https://github.com/aws/aws-cdk/blob/0653e6bead37ed92e47295010645009e3b97e246/packages/aws-cdk/bin/cdk.ts#L139

Reply:
Sure, the typing of `_` might be tricky.

But for example, `exclusively` can be added, and TSC knows the type of `argv` (that it has an `exclusively: boolean | undefined` field).

Reply:
Also if you wanted to force it to accept your definition of `_` you could do something like this:

```ts
const configuration = new Configuration({
  ...argv,
  _: argv._ as [Command, ...string[]]
});
```

(Which is a wacko type, btw. I didn't even know you could do that :) )